daml-lf/engine/src/test/scala/com/digitalasset/daml/lf/engine/InterfacesTest.scala

```diff
@@ -255,7 +281,16 @@ class InterfacesTest
     "be unable to exercise T1 (disguised as T2) by interface I2 via 'exercise by interface'" in {
       val command =
         ExerciseByInterfaceCommand(idI2, idT2, cid1, "C2", ValueRecord(None, ImmArray.empty))
-      run(command) shouldBe a[Left[_, _]]
+      run(command) match {
+        case Left(Error.Interpretation(err, _)) =>
```

Comment:
you might want to use `inside` to make the assertion a bit nicer.

Reply:
and for most of the other cases as well

Reply:
thanks for the suggestion!